cypress/integration/v2/blocks/feature/index.js

```diff
@@ -249,6 +249,11 @@ function styleTab( viewport, desktopOnly ) {
 		cy.waitFA()
 
 		desktopOnly( () => {
+			cy.adjust( 'Link / URL', 'https://www.google.com/' ).assertHtmlAttribute( '.ugb-button', 'href', 'https://www.google.com/', { assertBackend: false } )
+			cy.adjust( 'Open link in new tab', true ).assertHtmlAttribute( '.ugb-button', 'rel', 'noopener noreferrer', { assertBackend: false } )
```

Comment:
To make the assertion more specific, better to supply a regular expression instead of the complete string value e.g. `/noopener noreferrer/`